TTHAnalysis/python/plotter/ttH-multilepton/2lss_3l_plots_diff.txt

```diff
@@ -1,13 +1,27 @@
 # vim: syntax=sh
 2lep_2DpT_2bins: Hreco_pTHvis\:Hreco_pTHgen :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvis (GeV)",  YTitle="pTHgen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+#2lep_2DpT_2bins_withNu: Hreco_pTVisPlusNu\:Hreco_pTHgen :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvisplusNu (GeV)",  YTitle="pTHgen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+#2lep_2DpT_2bins_allgen: Hreco_pTHvis\:Hreco_pTTrueGenplusNu :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvis (GeV)",  YTitle="pTHgenall (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+
+2lep_2DpT_2bins_truegen: Hreco_pTHvis\:Hreco_pTTrueGen :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvis (GeV)",  YTitle="pTTrueGen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
 2lep_pTHvis_2bins                           :  if3(nLepFO>1, Hreco_pTHvis,-99) : [0,120,350];     XTitle="pT of reconstructed Higgs (GeV)"      , Legend='TL', MoreY=1.8, IncludeOverflows=False
 2lep_pTHvis_2bins_log                       :  if3(nLepFO>1, Hreco_pTHvis,-99) : [0,120,350];     XTitle="pT of reconstructed Higgs (GeV)"      , Legend='TL', MoreY=20, IncludeOverflows=False, Logy
 
+
 2lep_2DpT_3bins: Hreco_pTHvis\:Hreco_pTHgen :  [0,120,200,350]*[0,120,200,350] ;                  XTitle="pTHvis (GeV)",  YTitle="pTHgen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+2lep_2DpT_3bins_truegen: Hreco_pTHvis\:Hreco_pTTrueGen :  [0,120,200,350]*[0,120,200,350] ;                  XTitle="pTHvis (GeV)",  YTitle="pTTrueGen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+
```

Comment:
remove empty line addition and commented lines addition

Reply:
empty lines removal done.